doc/news/changes/minor/20171207DenisDavydov

```diff
@@ -0,0 +1,5 @@
+New: Improve support of triangular matrices in LAPACKFullMatrix wrappers.
+Add LAPACKFullMatrix::solve() to solve a system of equations either when
```

Comment:
Maybe say "Improve support of triangular matrices in LAPACKFullMatrix wrappers by adding a LAPACKFullMatrix::solve() functino that solves ..." to make the style more uniform with the other changelog entries...

Reply:
actually that `solve()` is NOT only for triangular, it now uses Cholesky also

Reply:
so it is related, but it not limited to improvements related to triangular matrices.